src/modules/manageTasks.js

```diff
@@ -0,0 +1,94 @@
+import {
+  taskName, dueDate, selectPriority, describeTask, taskBoxEdit, addForm,
+} from './documentObjects';
+import * as task from './addGroupName';
+
+const todoTask = () => ({
+  id: Date.now().toString(),
+  name: taskName.value,
+  dateDue: dueDate.value,
+  priority: selectPriority.value,
+  describe: describeTask.value,
+});
+
+export const renderTasks = (taskArray) => {
+  task.checkLocalStorage();
+  taskBoxEdit.innerHTML = '';
+  taskArray.forEach((task) => {
+    const htmlTask = `
+      <div class='task-box-div' id='tb-${task.id}'><h5 class='t-title' id='pt-${task.id}'><span>Title:</span> ${task.name}</h5></br>
+      <textarea class='t-describe'>${task.describe}</textarea><p class='t-priority'><span>Priority:</span> ${task.priority}</p>
+      <div class='edit-box'><p class='due-date'><span>Due Date:</span> ${task.dateDue}</p><div class='task-edit'><i class="fa fa-edit" id='${task.id}e'></i>
+      <i class='fa fa-trash-o' id='${task.id}d'></i>
+      </div></div></div></br>`;
+    taskBoxEdit.insertAdjacentHTML('afterbegin', htmlTask);
+  });
+};
```

Comment:
Dom manipulation and logic should be in separate modules. 😄 
![modules_requirement](https://user-images.githubusercontent.com/32361963/104726386-8d015700-5744-11eb-89d3-82b8639c72e0.png)

I can see you have done most of your dom manipulation in the index.js. Kindly use this file for all dom manipulation.